### DISCUSSION TITLE
PubPub outage?

### DISCUSSION THREAD

**Discussion by aimeediehl** [knowledgefutures/pubpub#2915] [Support]
Hi! PubPub won't load my site or my pub in progress this morning—it's been generating an application error message for the past 20 minutes or so—and the response from the helpdesk says it's closed on Fridays. Anyone else having this problem? And is there any way PubPub can send an automatic alert if a problem is detected?

Thanks and good luck to the PubPub team if you're troubleshooting ... we really appreciate everything you do.

**Comment by gabestein** [comment]
Yes, we're looking into it.

**Comment by gabestein** [comment]
Service should be restored.
